### ISSUE TITLE
Improve CI for releasing

### ISSUE THREAD

**Issue by borgmanJeremy** [flameshot-org/flameshot#966]
Some ideas for improving the CI system for releases:
1. Right now when I do a release I have to manually download the SHA and binaries for every type of release. I then manually check that all the binaries match the SHA's. It would be great if I could download 1 zip file that had all the binaries and SHA's computed in the CI. Then I would be able to easily verify the binaries without needing to do many separate manual downloads. 

1. It would be nice if the CI updated the VERSION info in the yaml files based on the current git tag. 

1. we should adjust some of the names so debian10, ubuntu 18.04 and ubuntu 20.04 have unique names.

**Comment by radoeka** [comment]
@borgmanJeremy is there a windows version of flameshot provided (I don't see it mentioned on the github flameshot project home page)?

**Comment by borgmanJeremy** [comment]
Yes it is included in the 0.8 release.

**Comment by mmahmoudian** [comment]
@radoeka the last bullet point of the release states:
> Improved Windows support

and also the `.msi` and `.zip` files are among the assets (they have the phrase `win32` and `win64`) in their file name.

Also the changelog on the website is also updated:
https://flameshot.js.org/#/changelog

**Comment by radoeka** [comment]
Found it.  Just overlooked this section "Prebuilt packages".  Should MS Windows be mentioned around that section of the home page (perhaps in the section called: Packages in the Repository)?

I'm helped, thanks.

**Comment by ZetaoYang** [comment]
@borgmanJeremy This PR https://github.com/flameshot-org/flameshot/pull/973 is only for your point 2 and 3.
For point 1, gathering separate packages together and compressing them into a single zip archive is difficult to achieve with github actions.

**Comment by borgmanJeremy** [comment]
Awesome thanks for the help!

**Comment by ZetaoYang** [comment]
@borgmanJeremy I found a compromise (gtihub actions artifacts) to compress all the packages together. I'm sorry, I used to think that public repositories were also 500MB storage, but today I double checked and public repositories are unlimited storage and unlimited computational capacity (runner minutes)! After that I will submit a new PR for point 1. So reopen this issue.

---
- https://poweruser.blog/storage-housekeeping-on-github-actions-e2997b5b23d1
